.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Pull request recommendations:
+
+- [ ] Name your pull request `your-development-type/short-description`.
+
+Ex:
+> feature/read-tiff-files
+
+or
+
+> bugfix/handle-file-not-found
+
+- [ ] If your pull request addresses a GitHub issue, link to it either in a commit message or in the PR description.
```

Comment:
Also I believe you can reference the ticket/PR with `#<number>` in the commit

Reply:
You are correct but meh not going to update it because it keeps in line with the bitbucket / jira style

Reply:
Just for reference 🙂 (we don't use bitbucket at my new place lol)